integration/oci_calls/oci_call_test.sh

```diff
@@ -76,14 +76,14 @@ function setup() {
 function run_oci_call() {
 	local -a oci_call=( "create" "start" "state" )
 
-	# This sleep is necessary to gather the correct logs
-	sleep 10
-
 	get_time
 
 	# Start a container
 	docker run -d --runtime=${RUNTIME} --name=${NAME} ${IMAGE} ${PAYLOAD}
 
+	# This sleep is necessary to gather the correct logs
+	sleep 10
```

Comment:
This is an awful lot of cpu cycles. Why does it take so long to gather these logs?

Reply:
just a thought: what about removing the `-d` from the docker run, so that we gather the log after the workload has already finish, that way I guess we wont have to `sleep`?

Reply:
@chavafg , we can not do that as part of the test is to run the container and then stop the container to gather the oci calls

Reply:
I had a read of the test - it's a little convoluted and not that easy to follow, but what I think happens is this part of the test launches the container and checks the correct launch calls were made. then a later part of the test kills the same container and checks the correct kill calls were made....

I agree the 10s feels like a long time. Is there any way we can poll, with a sleep/timeout loop, to check when the container is up and running - using a docker status command etc.?
Also, given the payload is just a busybox with a tail, it should not take more than 3-5s to get it running (ideally <2).
Still, if we know it might take upto 5s on some machines to launch, maybe 10s is not a terrible thing here...